src/Orckestra.Composer.MyAccount/Services/MembershipViewService.cs

```diff
@@ -154,7 +155,26 @@ public virtual async Task<LoginViewModel> LoginAsync(LoginParam param)
             var loginUrl = MyAccountUrlProvider.GetLoginUrl(new BaseUrlParameter { CultureInfo = param.CultureInfo, ReturnUrl = param.ReturnUrl });
             var userName = GenerateUserName(param.Username);
 
-            var loginResponse = Membership.LoginUser(userName, param.Password);
+            var loginResponse = false;
+            try
+            {
+                loginResponse = Membership.LoginUser(userName, param.Password);
+            }
+            catch (ComposerException e)
+            {
+                if (e.Errors[0].ErrorCode == "UserLockedDown")
+                {
+                    var customer = await CustomerRepository.GetCustomerByUsernameAsync(new GetCustomerByUsernameParam
+                    {
+                        Scope = param.Scope,
+                        Username = userName,
+                        CultureInfo = param.CultureInfo
+                    }).ConfigureAwait(false);
+                    e.Errors[0].Bag.Add("AccountLockedDownUntil", customer.AccountLockedDownUntil?.ToString("dd/MM/yyyy hh:mm tt"));
```

Comment:
is the value localized for different cultures?
How it will look for fr-CA?

You can  try define the format in localization General localization file, an example is 
![image](https://user-images.githubusercontent.com/6649972/195520060-cbcb6121-8cf7-4240-8e3f-4f1a8d2818c4.png)

or use somehow param.CultureInfo for correct formating